### ISSUE TITLE
Impossible to scroll content in Webview

### ISSUE THREAD

**Issue by rliffredo** [openhab/openhab-android#981]
### Actual behaviour
When the Webview displays a page with content exceeding the initial viewport (and requiring scrolling), it is not possible to scroll. In certain occasions it is possible to scroll a couple of lines, but not much more.  
As a result, it is impossible to see the full page content.

Note: I tested it only from internal lan, and not using proxies, vpn, or myopenhab.org.  
I believe the behavior should be about the same, since it is essentially about scrollbars.

### Expected behaviour
On BasicUI, the same Webview will correctly scroll, allowing to see the full page content.
 
### Steps to reproduce

Create a webview. The actual page should not fit into it.
```
sitemap global label="bug"
{
    Webview url="http://giallozafferano.it" height=20
}
```
The app will show only a part of the page, while in basic UI you can see the full content.

#### Client
* Android version: Reproduced on a tablet with android 5.0 and a phone with android 8.0
* Device model: Medion lifetab 8.9, Sony Xperia XZ compact
* Stock or customized ROM: Stock ROM, in both cases
* App version: 2.2.0
* Build flavor: Full
* Device language: English

#### Server
* Server version: 2.3
* Reverse Proxy: N/A
* Authentication method: None

**Comment by mueller-ma** [comment]
If scrolling is enabled for webviews and your webview is larger than your screen height, you wouldn't be able to scroll the sitemap view anymore. IMO the problem you describe doesn't seem like a bug to me rather a limitation. You could increase the height of the webview.

**Comment by rliffredo** [comment]
In the very same situation, on the very same device, Basic UI works.  
On the other hand, I agree that BasicUI has a different layout (leaving some space around) and scrolling is easier (you can always select the "frame", and then scroll that one).

**Comment by rliffredo** [comment]
After more investigation, it seems that this is creating some incompatibility with BasicUI:

- If I put `height=xx`, it will work correctly in BasicUI, but not work in android.
- If I do not put `height=xx`, then it will work correctly in android, but it might not work correctly in BasicUI.

This is visible in the example above: when removing the `height` attribute, BasicUI will simply display a very small frame.  
Note that increasing the height is not a solution, as you might not know in advance the actual height of the page.

**Comment by mueller-ma** [comment]
I created https://github.com/eclipse/smarthome/issues/6077. IMO the basic UI should auto adjust the height if no height attribute is set.

**Comment by mueller-ma** [comment]
@rliffredo Can you update to 2.3.6-beta (when it's available in Play/F-Droid) and set the height higher?

**Comment by rliffredo** [comment]
Yes, I will let you know as soon as there is an update in F-Droid.

**Comment by rliffredo** [comment]
OK, just tried, and I could not see any difference in behavior.
Tried with openHAB Beta `2.3.7-beta`

**Comment by mueller-ma** [comment]
The change in the Android app is that the webview now won't be larged than the actuall website content. You need to set a higher hight value for the webview.
